shipment_advice_planner/models/stock_picking.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    warehouse_id = fields.Many2one(
+        comodel_name="stock.warehouse", related="location_id.warehouse_id", store=True
+    )
+    can_be_planned_in_shipment_advice = fields.Boolean(
+        compute="_compute_can_be_planned_in_shipment_advice", store=True
```

Comment:
Why store=True? The search method is easy to implement

Reply:
Still not fan of having that value computed and stored on every picking while you could implement the search method returning a simple domain

Reply:
When you delete the shipment advice, planned_shipment_advice_id get emptied but the stored value does not get recomputed.

Anyway, this should not be stored, this is bad for the performance